span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/SpanFilter.java

```diff
@@ -55,6 +64,26 @@ public SpanFilter(Config config) {
       this.spanDropCriterion = parseStringList(criterion);
     }
 
+    if (config.hasPath(SPAN_DROP_FILTERS)) {
+      this.spanDropFilters =
+          config.getList(SPAN_DROP_FILTERS).stream()
+              .map(
+                  orFilters -> {
+                    List<HashMap<String, String>> andFilters =
+                        (List<HashMap<String, String>>) orFilters.unwrapped();
+                    return andFilters.stream()
+                        .map(
+                            filter ->
+                                new SpanDropFilter(
+                                    filter.get(TAG_KEY),
+                                    filter.get(OPERATOR),
+                                    filter.get(TAG_VALUE)))
+                        .collect(Collectors.toList());
+                  })
+              .collect(Collectors.toList());
+      LOG.info("Json String for Span drop criterion: {}", this.spanDropFilters);
```

Comment:
toString() missing in SpanDropFilter class

Reply:
Its not a Json right?

Reply:
No, it is List<List<SpanDropFilters>>, move the logging in the top before parsing in case of any failure we know what has failed.

Reply:
Updating this one...